python/dgl/data/utils.py

```diff
@@ -24,6 +25,22 @@ def _get_dgl_url(file_url):
     return repo_url + file_url
 
 
+def split_dataset(dataset, frac_list=[0.8, 0.1, 0.1], shuffle=False, random_state=42):
```

Comment:
Many people are against using mutable default value as an argument, e.g. [Using a mutable default value as an argument
](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html). I tend to agree them and maybe it's more secure to set None as default argument and do

```python
if frac_list is None:
    frac_list = [0.8, 0.1, 0.1]
```

Reply:
Done